lcls_tools/data_analysis/archiver.py

```diff
@@ -18,82 +20,97 @@
 TIMEOUT = 3
 
 
+@dataclass()
+class ArchiverData:
+    # todo make this not a union
```

Comment:
Can you add some comments here on the main goal of this data class? I think it's fairly obvious, but wouldn't hurt to write it down.